internal/rbd/nodeserver.go

```diff
@@ -145,11 +145,8 @@ func (ns *NodeServer) NodeStageVolume(
 	}
 	defer cr.DeleteCredentials()
 
-	if acquired := ns.VolumeLocks.TryAcquire(volID); !acquired {
```

Comment:
As its already discussed, I'm sure we all are aware of this, but just to remind again:

Locking is needed by Volume Healer, so at least for NodeStageVolume() we cannot remove it.
See: https://github.com/ceph/ceph-csi/pull/2108

Reply:
yeah, we are aware. One thing which I can do here is bringing back the nodestage and nodeunstage locks back.. let me refresh the PR with that.